sixlive/nova-text-copy-field
# Append ellipse only if text is truncated

## Description
When implementing this for files within our Nova app that are long values. I noticed that when the length is less than the truncated value passed to TextCopy it will append the ellipse (...) to the text regardless if it was truncated or not.

This checks the length of the text and determines if it is greater than truncate to then output the ellipse.

## Related Issues
None

## Related PRs
None

## Steps to Verify Functionality
If you have text length of (n) and truncate is set to (n-1) then you SHOULD see the ellipse (...) to the left of the Copy button.

If you have text length of (n) and truncate is set to (n) then you SHOULDN'T see the ellipse (...) to the left of the Copy button.
